integration_test/dashboard_test.dart

```diff
@@ -84,6 +88,8 @@ void main() {
     });
 
     testWidgets('test if seed backup warning disappears after checkbox is checked', (tester) async {
+      await api.initWallet(network: Network.Testnet);
```

Comment:
Does this mean that this test has a dependency on `testnet`, i.e. it actually requires an internet connection and fetches from a public node? 

I think it would be great if these tests would not depend on external things. Not super important for now.
On a more general note: For this test the actual seed really should not matter, why do we even need the api call here?

Reply:
maybe we could run 'signet' on this? (and not connect with any external infrastructure?). Alternatively, we could allow some custom `None` network where our wallet would be hardcoded to some value and not using any public infrastructure.

Reply:
> Does this mean that this test has a dependency on `testnet`, i.e. it actually requires an internet connection and fetches from a public node?

Yes it does, but its not different for any other test that is running the TenTenOneApp, as the wallet will be initialised there.  It's only done here explicitly, because the dashboard_test are running a different app config to be able to access the providers. 

> On a more general note: For this test the actual seed really should not matter, why do we even need the api call here?

Well, because the tests are failing otherwise due to a rust error. The seed is trying to access an uninitialised wallet. Once the logging PR is merged I will adapt and add an error handling.

Reply:
@da-kami https://github.com/itchysats/10101/pull/102/commits/451f23039bf77b019f914b33f5a358c2904dfda0